notebooks/bo_sphere.ipynb

```diff
@@ -108,39 +109,63 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "def logmap(x, x0):\n",
+    "    \"\"\"\n",
+    "    This functions maps a point lying on the sphere manifold into the tangent space of a second point of the manifold.\n",
```

Comment:
Probably
- "this function" instead of "this functions"
- "on the manifold" instead of "of the manifold"

Reply:
Done, thanks!